src/OgGroupAudienceHelper.php

```diff
@@ -17,8 +17,13 @@
 class OgGroupAudienceHelper {
 
   /**
+   * The default OG audience field name.
+   */
+  const DEFAULT_FIELD = 'og_group_ref';
```

Comment:
`og_group_ref` is an _old_ name I regret - I would love to change it. I'm just hesitant because of the implications it will have on upgrading existing installations :/